docker/ravendb-ubuntu/run-raven.sh

```diff
@@ -17,11 +14,51 @@ check_for_certificates() {
        [[ "$RAVEN_ARGS" == *"--Server.Certificate.Path"* ]] || \
        [[ "$RAVEN_ARGS" == *"--Server.Certificate.Load.Exec"* ]]; then
         RAVEN_SERVER_SCHEME="https"
-    else
-        RAVEN_SERVER_SCHEME="http"
     fi
 }
 
+# Determines the settings file location based on RAVEN_ARGS, using sed
+# \([^ ]*\) - captures the path value, stopping at the next space or the end
+# /\1/ - first path match, p - print, * - match anything around
+check_for_custom_settings_path() {
+    
+    CUSTOM_SETTINGS_PATH=""
+    if [[ "$RAVEN_ARGS" == *"-c "* ]]; then
+        # Extract the path after '-c'
+        CUSTOM_SETTINGS_PATH=$(echo "$RAVEN_ARGS" | sed -n 's/.*-c \([^ ]*\).*/\1/p')
+    elif [[ "$RAVEN_ARGS" == *"-c="* ]]; then
+        # Extract the path after '-c='
+        CUSTOM_SETTINGS_PATH=$(echo "$RAVEN_ARGS" | sed -n 's/.*-c=\([^ ]*\).*/\1/p')
+    elif [[ "$RAVEN_ARGS" == *"--config-path "* ]]; then
+        # Extract the path after '--config-path'
+        CUSTOM_SETTINGS_PATH=$(echo "$RAVEN_ARGS" | sed -n 's/.*--config-path \([^ ]*\).*/\1/p')
```

Comment:
what if there's more than one space between ?
```suggestion
        CUSTOM_SETTINGS_PATH=$(echo "$RAVEN_ARGS" | sed -n 's/.*--config-path     \([^ ]*\).*/\1/p')
```